### ISSUE TITLE
Tribler fresh install won't start

### ISSUE THREAD

**Issue by COUPON4Vitamins** [Tribler/tribler#6895]
Hi, I just installed Tribler and it won't start:

`liam@ideacentre:~$ tribler 
Starting Tribler...
/usr/share/tribler ~
Segmentation fault (core dumped)
`

/+o+-       liam@ideacentre
                  yyyyy- -yyyyyy+      OS: Ubuntu 22.04 jammy
               ://+//////-yyyyyyo      Kernel: x86_64 Linux 5.15.0-27-generic
           .++ .:/++++++/-.+sss/`      Uptime: 14h 28m
         .:++o:  /++++++++/:--:/-      Packages: 1836
        o:+o+:++.`..```.-/oo+++++/     Shell: bash 5.1.16
       .:+o:+o/.          `+sssoo+/    Resolution: 1920x1080
  .++/+:+oo+o:`             /sssooo.   DE: GNOME 41.4
 /+++//+:`oo+o               /::--:.   WM: Mutter
 \+/+o+++`o++o               ++////.   WM Theme: Adwaita
  .++.o+++oo+:`             /dddhhh.   GTK Theme: Yaru-blue-dark [GTK2/3]
       .+.o+oo:.          `oddhhhh+    Icon Theme: Yaru-blue
        \+.++o+o``-````.:ohdhhhhh+     Font: Ubuntu 11
         `:o+++ `ohhhhhhhhyo++os:      Disk: 164G / 1.7T (11%)
           .o:`.syhhhhhhh/.oo++o`      CPU: Intel Core i5-7400 @ 4x 3GHz
               /osyyyyyyo++ooo+++/     GPU: llvmpipe (LLVM 13.0.1, 256 bits)
                   ````` +oo+++o\:     RAM: 6459MiB / 7638MiB
                          `oo++.

**Comment by COUPON4Vitamins** [comment]
Sorry for the duplicate

**Comment by xoriole** [comment]
@COUPON4Vitamins Can you try this build?
https://jenkins-ci.tribler.org/job/Build-Tribler_release/job/Build-Ubuntu64/522/

**Comment by COUPON4Vitamins** [comment]
Thanks! This build works and loads faster than the previous working one.

**Comment by a-t-0** [comment]
I also confirm Tribler 7.11 does not work on Ubuntu 22.04. Based on the answer by @xoriole , I used the following temporary script to install tribler:
```sh
#!/bin/bash
install_tribler_from_source() {
    cd $git_dir
    git clone https://github.com/tribler/tribler
    conda deactivate
    yes | sudo apt install git libssl-dev libx11-6 libgmp-dev python3 python3-minimal python3-pip python3-libtorrent python3-pyqt5 python3-pyqt5.qtsvg python3-scipy
    pip3 install --upgrade -r tribler/requirements.txt
    pip install sentry-sdk
    chmod +x tribler/src/tribler.sh
    # Run tribler
    #/tribler/src/tribler.sh  > tribler.log
}
# Unstable performance, so this installation method is commented out.
#$(install_tribler_from_source)

install_tribler_from_snap() {
    sudo snap install tribler-bittorrent --beta

    # Remove snap install of Tribler:
    #sudo snap remove tribler-bittorrent --beta
}
# Beta is not safe for arbitrary user.
#$(install_tribler_from_snap)

install_tribler_from_deb() {
    #wget https://github.com/Tribler/tribler/releases/download/v7.11.0/tribler_7.11.0_all.deb
    #sudo apt-get install ./tribler_7.11.0_all.deb # Does not work.
    #sudo dpkg -i tribler_7.11.0_all.deb
    # Version 7.11 does not work on Ubuntu 22.04: https://github.com/Tribler/tribler/issues/6895#issuecomment-1122490517

    wget https://jenkins-ci.tribler.org/job/Build-Tribler_release/job/Build-Ubuntu64/522/artifact/tribler/build/debian/tribler_7.12.0-alpha.4-3-g0fb222e62_all.deb
    sudo dpkg -i tribler_7.12.0-alpha.4-3-g0fb222e62_all.deb
    # Launch Tribler in terminal by typing:
    # tribler

    # Remove tribler:
    #sudo dpkg -r tribler
}
$(install_tribler_from_deb)
```

**Comment by drew2a** [comment]
@xoriole is it fixed?

**Comment by xoriole** [comment]
Yes, the build is fixed so we'll need to do a release (7.12) to address this fully
